src/views/SystemWorkspace/components/TeamCard/index.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, IconExpandMore } from '@trussworks/react-uswds';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import { AvatarCircle } from 'components/shared/Avatar/Avatar';
+import teamRolesIndex from 'constants/teamRolesIndex';
+import { CedarRole } from 'queries/types/CedarRole';
+import { CedarAssigneeType } from 'types/graphql-global-types';
+import {
+  CedarRoleAssigneePerson,
+  UsernameWithRoles
+} from 'types/systemProfile';
+import getUsernamesWithRoles from 'utils/getUsernamesWithRoles';
+
+import SpacesCard from '../SpacesCard';
+
+function MemberRole({ person }: { person: UsernameWithRoles }) {
+  const { t } = useTranslation('systemWorkspace');
+  const p = person.roles[0];
+  const moreRolesCount = person.roles.length - 1;
+
+  return (
+    <div className="grid-col-6 desktop:grid-col-3 margin-bottom-1 display-flex">
+      <AvatarCircle
```

Comment:
There needs to be a bit more space between the initials icon and the name / role text